subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -427,9 +461,21 @@ void lll_conn_isr_tx(void *param)
 	/* assert if radio packet ptr is not set and radio started rx */
 	LL_ASSERT(!radio_is_ready());
 
+#if defined(CONFIG_BT_CTRL_DF_CONN_CTE_TX)
+	pdu_tx = get_last_tx_pdu(lll);
+	LL_ASSERT(pdu_tx);
+
+	if (pdu_tx->cp == PDU_DATA_CTE_PRESENT_BIT_ENABLED) {
+		cte_len = CTE_LEN_US(pdu_tx->cte_info.time);
+	} else {
+		cte_len = 0;
+	}
+#endif /* CONFIG_BT_CTRL_DF_CONN_CTE_TX */
 	/* +/- 2us active clock jitter, +1 us hcto compensation */
-	hcto = radio_tmr_tifs_base_get() + EVENT_IFS_US + 4 +
-		RANGE_DELAY_US + 1;
+	hcto = radio_tmr_tifs_base_get() + EVENT_IFS_US + 4 + RANGE_DELAY_US + 1;
```

Comment:
There seems to be quite a few lines in this PR that has only been reformatted and haven't really changed. I suggest that you avoid doing that, so that your PRs will be more clear and easier to review, as well as given a clearer git history :)

Reply:
There are some lines that were just re-formatted. I didn't catch them, I'll put more attention to that in future.

Reply:
Yes it is. I've updated it to: `(EVENT_CLOCK_JITTER_US << 1)` and the `+ 1` at the end to: `HCTO_START_DELAY_US`